package.json

```diff
@@ -1,51 +1,45 @@
 {
   "name": "@teamleader/ui-icons",
   "description": "Teamleader UI icons",
-  "version": "0.2.11",
+  "version": "0.2.12",
   "author": "Teamleader <development@teamleader.eu> (https://www.teamleader.eu)",
   "bugs": {
     "url": "https://github.com/teamleadercrm/ui-icons/issues"
   },
+  "dependencies": {},
   "devDependencies": {
-    "@balajmarius/svg-to-jsx": "^1.0.8",
-    "babel-eslint": "^8.0.2",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.3.3",
+    "@babel/plugin-proposal-export-default-from": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.0.0",
+    "@balajmarius/svg2jsx": "^1.0.8",
     "bluebird": "^3.5.1",
     "cheerio": "^1.0.0-rc.2",
     "cli-color": "^1.2.0",
-    "eslint": "^4.11.0",
-    "eslint-config-prettier": "^2.7.0",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-config-standard-react": "^5.0.0",
-    "eslint-plugin-babel": "^4.1.2",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-promise": "^3.6.0",
-    "eslint-plugin-react": "^7.4.0",
-    "eslint-plugin-standard": "^3.0.1",
     "fs-extra": "^4.0.2",
     "glob": "^7.1.2",
     "lodash.camelcase": "^4.3.0",
     "lodash.findkey": "^4.6.0",
     "lodash.upperfirst": "^4.3.1",
-    "prettier": "^1.8.2"
+    "prettier": "^1.16.4"
   },
   "engines": {
     "node": "^8.0.0",
     "npm": "^5.0.0"
   },
   "homepage": "https://github.com/teamleadercrm/ui-icons#readme",
+  "jsnext:main": "es/index.js",
```

Comment:
I did some extra reading and writing on the topic and I think we can safely remove this one and go for module alone.